### ISSUE TITLE
Cannot perform "refine tracklets" on maDLC

### ISSUE THREAD

**Issue by quirinkrabi** [DeepLabCut/DeepLabCut#1500]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Dear developers,

I am using the latest deeplabcut from github in an Anaconda environment, with GPU (all working fine it seems), using the deeplabcut GUI to do maDLC.

I successfully performed maDLC network training on a video of two rats interacting (one white one dark), and I successfully analyzed the same video afterwards to test the tracking, and converted this to tracklets.

I got various pickle files as outputs.

Then on the "Refine tracklets" tab, I performed "Create tracks". I got the ..._el.h5 file as output.

Then I wanted to perform, on the same tab, the "Optional: Refine tracks" function.

This however fails with an error.

(Please see below, where I pasted all from "Step 1: Analyze Videos" onwards).

Would you kindly have any suggestion how to solve this? Or am I overlooking something?

Best wishes,
Quirin

### Operating System

Windows 10

### DeepLabCut version

dlc version 2.2.0.1

### DeepLabCut mode

multi animal

### Device type

gpu: Nvidia Quadro T1000

### Relevant log output

```shell
Running  DLC_dlcrnetms5_test4Sep1shuffle1_180000  with # of trainingiterations: 180000
Activating extracting of PAFs
Analyzing data...
40it [00:03, 10.29it/s]
Results for 180000 training iterations: 95, shuffle 1:
Train error: 4.98 pixels. Test error: 10.67 pixels.
With pcutoff of 0.6:
Train error: 4.98 pixels. Test error: 10.67 pixels.
##########################################
Average Euclidean distance to GT per individual (in pixels)
individuals
Stimulusrat    3.575547
Testrat        7.179212
Average Euclidean distance to GT per bodypart (in pixels)
bodyparts
leftear      5.785083
rightear     5.844563
shoulder     9.230907
snout       11.160438
spine1       4.743156
spine2       5.251940
spine3       2.800900
spine4       2.728229
tail1        3.021923
tail2        5.815999
tailbase     2.696431
tailend      2.645095
Done and results stored for snapshot:  snapshot-180000
Selecting best skeleton...
Graph 1|10
40it [00:00, 791.65it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 3957.17it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 3311.79it/s]
Graph 2|10
40it [00:00, 566.04it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 3956.42it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 3310.09it/s]
Graph 3|10
40it [00:00, 440.04it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 3958.94it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 4996.04it/s]
Graph 4|10
40it [00:00, 363.79it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 2557.89it/s]
100%|██████████████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<?, ?it/s]
Graph 5|10
40it [00:00, 344.96it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 2565.91it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 2560.55it/s]
Graph 6|10
40it [00:00, 290.59it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 2561.02it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 1703.22it/s]
Graph 7|10
40it [00:00, 242.64it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 4983.43it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 3316.97it/s]
Graph 8|10
40it [00:00, 188.60it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 3307.68it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 3954.47it/s]
Graph 9|10
40it [00:00, 197.37it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<00:00, 3999.34it/s]
100%|██████████████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<?, ?it/s]
Graph 10|10
40it [00:00, 185.28it/s]
100%|██████████████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<?, ?it/s]
100%|██████████████████████████████████████████████████████████████████████████████████████████| 40/40 [00:00<?, ?it/s]
C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01/evaluation-results/  already exists!
C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01\evaluation-results\iteration-0\test4Sep1-trainset95shuffle1  already exists!
C:\ANACONDA\Anaconda3\envs\DEEPLABCUT\lib\site-packages\tensorflow\python\keras\engine\base_layer_v1.py:1694: UserWarning: `layer.apply` is deprecated and will be removed in a future version. Please use `layer.__call__` method instead.
  warnings.warn('`layer.apply` is deprecated and '
Activating extracting of PAFs
Analyzing data...
3it [00:00,  9.88it/s]
Saving plots...
C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01\evaluation-results\iteration-0\test4Sep1-trainset95shuffle1\maps  already exists!
100%|████████████████████████████████████████████████████████████████████████████████████| 3/3 [00:05<00:00,  1.77s/it]
Analyzing ...
Using snapshot-180000 for model C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01\dlc-models\iteration-0\test4Sep1-trainset95shuffle1
Activating extracting of PAFs
Starting to analyze %  C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01\Gq-II_OTR-Cre_vlPAG_01_SI_salineshortcropped.avi
C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01  already exists!
Loading  C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01\Gq-II_OTR-Cre_vlPAG_01_SI_salineshortcropped.avi
Duration of video [s]:  60.0 , recorded with  25.0 fps!
Overall # of frames:  1500  found with (before cropping) frame dimensions:  266 286
Starting to extract posture
100%|██████████████████████████████████████████████████████████████████████████████| 1500/1500 [00:23<00:00, 63.06it/s]
Saving results in C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01...
The videos are analyzed. Time to assemble animals and track 'em...
 Call 'create_video_with_all_detections' to check multi-animal detection quality before tracking.
If the tracking is not satisfactory for some videos, consider expanding the training set. You can use the function 'extract_outlier_frames' to extract a few representative outlier frames.
Creating labeled video for  Gq-II_OTR-Cre_vlPAG_01_SI_salineshortcropped
100%|█████████████████████████████████████████████████████████████████████████████| 1500/1500 [00:06<00:00, 243.92it/s]
Using snapshot-180000 for model C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01\dlc-models\iteration-0\test4Sep1-trainset95shuffle1
Processing...  C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01\Gq-II_OTR-Cre_vlPAG_01_SI_salineshortcropped.avi
C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01  already exists!
Analyzing C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01\Gq-II_OTR-Cre_vlPAG_01_SI_salineshortcroppedDLC_dlcrnetms5_test4Sep1shuffle1_180000.h5
1500it [00:03, 436.39it/s]
1500it [00:04, 351.79it/s]
The tracklets were created. Now you can 'refine_tracklets'.
Processing...  C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01\Gq-II_OTR-Cre_vlPAG_01_SI_salineshortcropped.avi
C:\ANACONDA\DATAANALYSIS\DeepLabCut-projects\test4-Quirin-2021-09-01  already exists!
100%|█████████████████████████████████████████████████████████████████████████████████| 50/50 [00:00<00:00, 500.37it/s]
C:\ANACONDA\Anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\refine_training_dataset\stitch.py:656: UserWarning: No optimal solution found. Employing black magic...
  warnings.warn("No optimal solution found. Employing black magic...")
Traceback (most recent call last):
  File "C:\ANACONDA\Anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\gui\refine_tracklets.py", line 313, in refine_tracklets
    self.manager, self.viz = deeplabcut.refine_tracklets(
  File "C:\ANACONDA\Anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\gui\tracklet_toolbox.py", line 793, in refine_tracklets
    manager.load_tracklets_from_hdf(pickle_or_h5_file)
  File "C:\ANACONDA\Anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\refine_training_dataset\tracklets.py", line 238, in load_tracklets_from_hdf
    df = pd.read_hdf(filename)
  File "C:\ANACONDA\Anaconda3\envs\DEEPLABCUT\lib\site-packages\pandas\io\pytables.py", line 427, in read_hdf
    raise FileNotFoundError(f"File {path_or_buf} does not exist")
FileNotFoundError: File  does not exist
```

### Anything else?

I will gladly provide any further information required. Please note that I am a beginner with programming and command line usage...

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by quirinkrabi** [comment]
I wonder if the problem could be that something is wrong with the GUI. See below a screenshot of how it looks in my case.
![grafik](https://user-images.githubusercontent.com/89841165/132723729-dd6c5c87-3756-4f0a-a1b5-9185b51a6f52.png)

However, if I go onto "help", it says that there should be a parameter to set the direction of the .h5 file.
![grafik](https://user-images.githubusercontent.com/89841165/132724060-b3796ab7-5507-48e6-9208-8245087cd4ef.png)

This clearly is not in the GUI that I am seeing...

**Comment by backyardbiomech** [comment]
This is a duplicate of #1489 and is fixed in #1492 which isn't in a release version yet. You can [install the latest code from GitHub](https://deeplabcut.github.io/DeepLabCut/docs/recipes/installTips.html), or you can call refine tracklets on the command line, where you can pass the pickle or h5 file directly.

**Comment by quirinkrabi** [comment]
Sorry, I had not found it. My bad. Thank you for pointing this out, I will give it a try with the GitHub installation (I thought I had already done this but apparently I did something wrong).
